brion/plugin/morphologyHDF5.cpp

```diff
@@ -760,6 +760,7 @@ bool MorphologyHDF5:: _readV2Metadata() const
     {
         detail::SilenceHDF5 silence;
         _file.openGroup( _g_root );
+        _version = MORPHOLOGY_VERSION_H5_2;
```

Comment:
This means it's possible to have v2 morphology files without the version attribute?